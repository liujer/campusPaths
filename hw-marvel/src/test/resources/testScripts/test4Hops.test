# Tests if path can be found in four hops between two
# nodes
LoadGraph testFour testTSV.tsv

FindPath testFour one seven

FindPath testFour seven one
