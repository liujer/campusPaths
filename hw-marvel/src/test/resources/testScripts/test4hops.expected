# Tests if path can be found in four hops between two
# nodes
loaded graph testFour

path from one to seven:
one to two via e12
two to five via e25
five to six via e56
six to seven via e67

path from seven to one:
seven to six via e67
six to five via e56
five to two via e25
two to one via e12